Dockerfile

```diff
@@ -6,7 +6,7 @@ WORKDIR /dradis
 
 # Install dependencies for Ruby build and gems
 RUN apt-get update -qq && \
-apt-get install --no-install-recommends -y curl git libjemalloc2 libvips sqlite3 && \
+apt-get install --no-install-recommends -y curl git libjemalloc2 libvips sqlite3 redis-server && \
```

Comment:
I know it's silly, but I'd split the regis-server into another apt-line.

It's likely we'll end up splitting this in a separate container, in the future or in Pro, and that would make the difference clearer.